packages/multichain-account-service/src/types.ts

```diff
@@ -122,3 +130,7 @@ export type MultichainAccountServiceMessenger = RestrictedMessenger<
   AllowedActions['type'],
   AllowedEvents['type']
 >;
+
```

Comment:
Added the newly added type alias from the accounts-api just for testing purposes, will swap out once I cut a release.